app/templates/administrative-units/administrative-unit/legal-structures/legal-structure/index.hbs

```diff
@@ -0,0 +1,51 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <PageHeader>
+      <:title>Verbonden juridische structuur: {{@model.associatedStructure.name}}</:title>
+      <:subtitle>{{@model.administrativeUnit.name}}</:subtitle>
+      {{! TODO: Replace this with the "edit" route once it is created }}
+{{!--       <:action>
+        <AuLink
+          @route="administrative-units.administrative-unit.legal-structures.legal-structure.edit"
+          @skin="button-secondary"
+          @icon="pencil"
+          @iconAlignment="left"
+        >
+          Bewerk
+        </AuLink>
+      </:action> --}}
+    </PageHeader>
+
+    <DataCard>
+      <:title>Contactgegevens</:title>
+      <:card as |Card|>
+        <Card.Columns>
+          <:left as |Item|>
+            <Item>
+              <:label>Naam</:label>
+              <:content>{{@model.associatedStructure.name}}</:content>
+            </Item>
+            <Item>
+              <:label>Locatie</:label>
+              <:content>{{@model.associatedStructure.address.fullAddress}}</:content>
+            </Item>
+            <Item>
+              <:label>Provincie</:label>
+              <:content>{{@model.associatedStructure.address.province}}</:content>
```

Comment:
This seems to be empty in the dummy data. Not sure if this will be a required field but I would wrap it in an if check.

Reply:
Yes I didn't add it in the dummy data because it wasn't in the fields of the index list page. An if check should be good.